.github/workflows/canary-integration-test.yml

```diff
@@ -34,12 +34,17 @@ jobs:
         kubectl cluster-info
         kubectl get pods -n kube-system
 
-    - name: use local disk
+    - name: use local disk and create partitions for osds
       run: |
+        BLOCK=$(sudo lsblk --paths|awk '/14G/ {print $1}'| head -1)
+        BLOCK_DATA_PART=${BLOCK}1
+        sudo dmsetup version||true
         sudo swapoff --all --verbose
         sudo umount /mnt
         # search for the device since it keeps changing between sda and sdb
-        sudo wipefs --all --force /dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1
+        sudo wipefs --all --force "$BLOCK_DATA_PART"
+        sudo lsblk
+        tests/scripts/create-bluestore-partitions.sh --disk "$BLOCK" --osd-count 2
```

Comment:
By creating partitions in the test setup that have bluestore info on them, are we not testing some parts of Rook OSD provisioning which would otherwise add bluestore info onto partitions?

Reply:
I think I understand, to address your concern let me perhaps add a dedicated scenario for 2 disks? So I'll leave the current model with one disk accessing a raw drive and a new one with 2 disks on partitions. Would that work for you?